changelog.md

```diff
@@ -1,5 +1,111 @@
+  * Improve packaging support ([#1173](https://github.com/SFML/SFML/pull/1173))
+  * Added Tagfile generation and search ([#1327](https://github.com/SFML/SFML/pull/1327))
+  * Set policy CMP0042 if cmake version > 3.0 ([#1206](https://github.com/SFML/SFML/pull/1206))
+  * Added CMake variables to select the modules to be built ([#800](https://github.com/SFML/SFML/pull/800))
+  * Updated the binary libs from exlibs/libs-ios in order to work with Xcode 8 and iOS 10 ([#1209](https://github.com/SFML/SFML/pull/1209))
+  * Do not install extlibs if `SFML_USE_SYSTEM_DEPS` is true ([#1237](https://github.com/SFML/SFML/pull/1237))
+  * Updated stb_image to v2.16 and stb_image_write to v1.07 ([#1270](https://github.com/SFML/SFML/pull/1270))
+  * Fixed various type conversion/comparison warnings ([#1325](https://github.com/SFML/SFML/pull/1325))
+  * [macOS] Fixed launch of cocoa examples ([#1334](https://github.com/SFML/SFML/pull/1334))
+  * [macOS] Improved application signing process ([#1194](https://github.com/SFML/SFML/pull/1194))
+  * [macOS] Improve cmake script ([#1215](https://github.com/SFML/SFML/pull/1215))
+  * [macOS] Use `-stdlib=libc++` ([#1361](https://github.com/SFML/SFML/pull/1361))
+  * [macOS] Fix CMake warning since CMake 3.9 ([#1371](https://github.com/SFML/SFML/pull/1371))
+  * [iOS] Use a CMake toolchain file for iOS build ([#1269](https://github.com/SFML/SFML/pull/1269))
+  * [iOS] Install extlibs if needed ([#1348](https://github.com/SFML/SFML/pull/1348))
+  * [iOS] Drop 32 bit support ([#1374](https://github.com/SFML/SFML/pull/1374))
+  * [iOS] Force correct iOS architecture for cmake ([#1377](https://github.com/SFML/SFML/pull/1377))
+  * [iOS] Find OpenAL correctly ([#1376](https://github.com/SFML/SFML/pull/1376))
+  * [Android] Increased minimum API version to 14 ([#1362](https://github.com/SFML/SFML/pull/1362))
+  * [Android] Removed custom toolchain and added support for the newest NDK version ([#1350](https://github.com/SFML/SFML/pull/1350))
+
+### System
+
+**Bugfixes**
+
+  * Fixed crash when `sf::Clock` is constructed in a global scope ([#1258](https://github.com/SFML/SFML/pull/1258))
+
+### Window
+
+**Features**
+
+  * Implemented Cursor API ([#827](https://github.com/SFML/SFML/pull/827))
+  * Implemented Clipboard API ([#1221](https://github.com/SFML/SFML/pull/1221))
+  * [Windows] Added support for interfacing with joysticks via DirectInput when it is available ([#1326](https://github.com/SFML/SFML/pull/1326))
+
+**Bugfixes**
+
+  * Updated documentation ([#1255](https://github.com/SFML/SFML/pull/1255), [#1310](https://github.com/SFML/SFML/pull/1310))
+  * [macOS] Fixed window menu not working ([#1193](https://github.com/SFML/SFML/pull/1193))
+  * [macOS] Removed unused variable ([#1276](https://github.com/SFML/SFML/pull/1276))
```

Comment:
Refactors don't need to be in the changelog unless they have an impact on the API